components/build/openshift-pipelines/allow-argocd-create-sa.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: allow-argo-cd-sa
```

Comment:
not really associated with this PR

Reply:
This enables us to reduce the blanket cluster-admin which the cluster config argocd has.